src/index.ts

```diff
@@ -1 +1,9 @@
-// hello world
+/**
+ * Create and Run the server
+ */
+
+import { App } from "./App";
+import { MockedProxyAPIApp } from "./mocks/MockedProxyAPIApp";
+
+new MockedProxyAPIApp().startServer();
```

Comment:
You may want to add a TODO here comment (we will get rid of the mocked proxy once real API are open). 
As a general advice, always add a TODO comment to help the reviewers catching temporary solutions.